parser_test.go

```diff
@@ -3153,9 +3283,9 @@ func TestCreateTable(t *testing.T) {
 		},
 		{
 			name:         "create table default",
-			stmt:         "CREATE TABLE t (a INT CONSTRAINT default_constraint DEFAULT 0, b INT DEFAULT -1.1, c INT DEFAULT 0x1, d TEXT DEFAULT 'foo', e TEXT DEFAULT ('foo'));",
-			deparsed:     "CREATE TABLE t (a INT CONSTRAINT default_constraint DEFAULT 0, b INT DEFAULT -1.1, c INT DEFAULT 0x1, d TEXT DEFAULT 'foo', e TEXT DEFAULT ('foo'))",
-			expectedHash: "26c558fd4e4dfb28a9bd399d7872bd24576214a0676eb4a7d3f97362734a03d9",
+			stmt:         "CREATE TABLE t (a INT CONSTRAINT default_constraint DEFAULT 0, b INT DEFAULT -1.1, c INT DEFAULT 0x1, d TEXT DEFAULT 'foo', e TEXT DEFAULT ('foo'), f INT DEFAULT +1);",
```

Comment:
Test for state 193. We can provide a signed numeric literal to a `DEFAULT` constraint.